src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.Objects;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static RegistrationService registrationService;
+    private static User firstUser;
+    private static User secondUser;
+    private static User thirdUser;
+    private static User fourthUser;
+    private static User fifthUser;
+    private static User sixthUser;
+    private static User seventhUser;
+    private static User eighthUser;
+    private static User ninthUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        firstUser = new User("Bob", "qwe123", 55);
+        secondUser = new User("Alice", "qwe123", 17);
+        thirdUser = new User("Bob", "567123", 18);
+        fourthUser = new User("John", "qwe56", 19);
+        fifthUser = new User("Max", "qwe569", 33);
+        sixthUser = new User(null, "qwe569", 33);
+        seventhUser = new User("", "qwe569", 33);
+        eighthUser = new User("Ivan", "", 33);
+        ninthUser = new User("Alex", null, 33);
+    }
+
+    @BeforeEach
+    void userRegister_Ok() {
+        Storage.people.clear();
+        registrationService.register(firstUser);
+        User registerUser = new User("Bob", "qwe123", 55);
+        boolean actual = Objects.equals(registerUser, Storage.people.get(0));
+        assertTrue(actual);
+    }
+
+    @Test
+    void repeatingName_NotOk() {
```

Comment:
```suggestion
    void register_repeatingName_NotOk() {
```

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: <methodUnderTest>_<state>_<expectedBehavior>; For example, if we are testing the method register with a null